### ISSUE TITLE
PyPDF4 is dead

### ISSUE THREAD

**Issue by MartinThoma** [claird/PyPDF4#103]
PyPDF4 is dead:

* [Last release on PyPI](https://pypi.org/project/PyPDF4/#history): 2018
* Last commit in this repo:  2020
* Last response to an issue: ... I couldn't find any

I suggest to do one of the following:

* a) Delete it from PyPI + delete this repository
* b) Add a note at the top of the README that the project is no longer active + add the trove classifier [`Development Status :: 7 - Inactive`](https://pypi.org/classifiers/) + make a "final" release on PyPI + Archive the github repository

**Comment by ghost** [comment]
Agreed. I would prefer to see development focussed on PyPDF2 and there is an issue discussing its future here: https://github.com/mstamy2/PyPDF2/issues/657

**Comment by pubpub-zz** [comment]
I've produced a fork with new features/corrections.
I did not had time yet to publish it further. What's your opinion about it ?
https://github.com/pubpub-zz/PyPDF4

**Comment by MartinThoma** [comment]
It's awesome when people like you put their time into such projects!

We have a couple of pdf libraries already and I have to think about xkcd: 
![image](https://user-images.githubusercontent.com/1658117/160913228-70b14ff5-f178-42ec-b355-c289caea7126.png)

I would like it a lot more if we had less pdf projects with a bigger community behind them

**Comment by pubpub-zz** [comment]
> It's awesome when people like you put their time into such projects!
> 
> We have a couple of pdf libraries already and I have to think about xkcd: ![image](https://user-images.githubusercontent.com/1658117/160913228-70b14ff5-f178-42ec-b355-c289caea7126.png)
> 
> I would like it a lot more if we had less pdf projects with a bigger community behind them

pyPdf4 was the most advanced/active mid 20, this is why I worked on. My 'Fork' is a set of fixes/improvement,I've proposed as contribution to @claird,like as some other did.
You're proposal to delete this fork which.
If you are willing to help all of us to keep alive pypdf4, I will be personnally very happy 😄

**Comment by MartinThoma** [comment]
PyPDF2 is back!

I've just got the permissions on https://github.com/mstamy2/PyPDF2 and on pypi for PyPDF2.

I would love if we could unite the various pdf packages and build something great together

Some issues you might be interested in:

- https://github.com/mstamy2/PyPDF2/issues/657
- https://github.com/mstamy2/PyPDF2/issues/658
- https://github.com/mstamy2/PyPDF2/issues/659

I'm also thinking about moving the projects in the github organization https://github.com/py-pdf. That could especially be interesting if we want to merge PyPDF2 / pypdf3 into one project in the long run.

**Comment by clach04** [comment]
My 2 cents, I would really hate to see this repo/forked deleted. If someone is relying on it deleting it and forcing people to update working code to now work with PyPDF2 is a needless overhead. I'm grateful to all the forks for the contributions made (in spite of the difficulty in finding the one true PDF lib ;)) and I'm really happy to hear @MartinThoma is organizing a sustainable maintained version of PyPDF.

My preference would be to archive this repo, ideally with a readme update. It's clearly marked as a fork of PyPDF2 so even if it is not archived/updated it is easy for people to discover the upstream maintained version :-)
